src/store/actions.js

```diff
@@ -537,6 +539,24 @@ export default {
     // torus.updateStaticData({ isUnlocked: true })
     dispatch('cleanupOAuth', { oAuthToken })
   },
+  async getUserDapps({ commit, dispatch, state }, { selectedAddress }) {
+    try {
+      const response = await get(`${config.developerDashboardUrl}/projects/user-projects?chain_namespace=evm&public_address=${selectedAddress}`)
```

Comment:
Do this in end.vue pls
Only use tkeyPrivKey to derive this

Reply:
I saw that in End.vue it'll send all the keys including tkey to OpenLoginHandler via broadcast channel. Then keys are passed to the action `handleLogin`. So I think we can derive keys in action as well.